tests/vmi_multus_test.go

```diff
@@ -152,10 +154,12 @@ var _ = Describe("Multus", func() {
 	})
 
 	Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance using different types of interfaces.", func() {
+		CIDR1 := "10.1.1.1/24"
```

Comment:
I still don't think that reusing this variable in the ptp tests makes sense. The CIDR of the ptp is not "10.1.1.1/24" it is "10.1.1.0/24".
One of the main reasons for reusing is that if you will change the value, you will need to change only one place.
In this case, if you will change `CIDR1` to equal "10.1.1.7/24" for example. It will break the ptp tests.

Reply:
OK. will return it to the hard coded address in the ptp context

Reply:
If you wish, on a separate PR, you can remove the hardcoded subnet from `ptpConfCRD` and extract it to a variable.
Then on the ptp tests add a gateway (the address the tests are pinging to) parameter calculated in the [following](https://github.com/containernetworking/cni/blob/6737bc8207fd58727a46bba2cfb74f4e9391ad4f/plugins/ipam/host-local/allocator.go#L78) way.